packages/components/src/components/expandable/expandable.ts

```diff
@@ -0,0 +1,193 @@
+import { css, html } from 'lit';
+import { customElement } from '../../../src/internal/register-custom-element';
+import { property, query } from 'lit/decorators.js';
+import { watch } from '../../internal/watch.js';
+import componentStyles from '../../styles/component.styles';
+import cx from 'classix';
+import SolidElement from '../../internal/solid-element';
+
+/**
+ * @summary Expandable shows a brief summary and expands to show additional content.
+ * @documentation https://solid.union-investment.com/[storybook-link]/expandable
+ * @status stable
+ * @since 1.1
+ *
+ * @dependency sd-icon
+ *
+ * @slot - Content of the expandable
+ *
+ * @event sd-show - Emitted when the expandable opens.
+ * @event sd-after-show - Emitted after the expandable opens and all animations are complete.
+ * @event sd-hide - Emitted when the expandable closes.
+ * @event sd-after-hide - Emitted after the expandable closes and all animations are complete.
+ *
+ * @csspart content - The content of the expandable.
+ *
+ * @cssproperty --component-expandable-max-block-size - Different value for initial visible block (default: 90px)
+ */
+@customElement('sd-expandable')
+export default class SdExpandable extends SolidElement {
+  @query('.shrinker') shrinker: HTMLElement;
+  @query('details') details: HTMLDetailsElement;
+
+  /**
+   * Used to check whether the component is expanded or not.
+   */
+  @property({ type: Boolean, reflect: true }) open = false;
+
+  /** Inverts the expandable and sets the primary color background. */
+  @property({ type: Boolean, reflect: true }) inverted = false;
+
+  /** Controls the height of the gradient depending on if it is used with `paragraph` or `leadtext`. */
```

Comment:
I had a look on the code and it changes the height of the gradient from 24px (paragraph) to 32px (leadtext). Unfortunately it's not very noticeable. Maybe adding the values to the description helps a bit.

Reply:
I just updated the docs. Like you mention, this feels like a very small change. This is part of the code that I was handed over, but it might be worth discussing whether or not we need it in the first place.